src/libopensc/card-cardos.c

```diff
@@ -781,6 +788,8 @@ cardos_set_security_env(sc_card_t *card,
 	if (card->type == SC_CARD_TYPE_CARDOS_CIE_V1) {
 		cardos_restore_security_env(card, 0x30);
 		apdu.p1 = 0xF1;
+	} else if (card->type == SC_CARD_TYPE_CARDOS_V5_3) {
+		apdu.p1 = 0x41;
```

Comment:
If I'm reading the 5.0 user manual correctly, then this change should also apply, because otherwise we would also need a usage qualifier for 5.0 (which we aren't currently adding). Unfortunately, I don't have an actual card. @szikora could you have a look?